### ISSUE TITLE
Title field showed as last field when creating new content

### ISSUE THREAD

**Issue by 1nf1n7y** [signalpoint/DrupalGap#738]
I am using title filed in the content type. but when create new content it exist as a last item with "undefined" label . this problem only inside drupalgap not in drupal website.

 in drupal website:
![title 2](https://cloud.githubusercontent.com/assets/9547879/13398913/979d62b8-df11-11e5-911f-3b24c5e9442e.JPG)

inside drupalgap:
![title filed](https://cloud.githubusercontent.com/assets/9547879/13398920/a0002e54-df11-11e5-9ad3-ef4ed0a44590.JPG)

**Comment by signalpoint** [comment]
@ahmedbarwary You should be able to use `hook_form_alter()` in DrupalGap to set the element's `weight' on the title to zero, and it should float to the top.

**Comment by 1nf1n7y** [comment]
@signalpoint  thank you . when I tried to changed the value of 'weight' to zero but nothing happend:
`form.elements['title_field'].weight = 0;`

**Comment by signalpoint** [comment]
@ahmedbarwary Thank you for following up. It's probable that `title_field` isn't the correct name to use there. It would be `title` for sites without the Title module installed, otherwise I think it'd be `field_title` if the Title module is installed. You can verify the name of the element by doing a `console.log(form.elements);` in your form builder/alter.

**Comment by 1nf1n7y** [comment]
I verified that the name of the title field is 'title_field' and the title module is installed in both drupal and drupalgap. this is the console log for the form.elements:
**field_client_name: Object
field_email: Object
field_med_photo: Object
field_phone: Object
language: Object
nid: Object
submit: Object
title: false
title_field: Object**
description: ""
field_info_field: Object
field_info_instance: Object
is_field: true
name: "title_field"
required: 1
title: "أسم العلاج"
type: "text"
und: Object
weight: 0

**Comment by g33kyg1rl** [comment]
I am also running into this on my drupalgap installation. Anything I can do to assist this being resolved? :)

**Comment by signalpoint** [comment]
The `_drupalgap_form_render_elements` function here is most likely where the problem lies, I'm guessing it has to do with whatever element(s) get a weight of zero, or how we handle converting negative weights.

**Comment by g33kyg1rl** [comment]
Yes, I was able to confirm the issue is with the negative weights.
